src/app/search/search-filters.ts

```diff
@@ -1183,6 +1188,35 @@ function searchFilters(
       hasLight(item: DimItem) {
         return item.primStat && statHashes.has(item.primStat.statHash);
       },
+      curated(item: D2Item) {
+        if (!item) {
+          return false;
+        }
+
+        const socketLength = Number(
+          item.sockets && item.sockets.sockets && item.sockets.sockets.length - 2 // ignore trackers
+        );
+        let oneSocketPerPlug = true;
+        for (let i = 0; i < socketLength; i++) {
+          oneSocketPerPlug =
+            oneSocketPerPlug &&
+            Boolean(
+              item.sockets &&
+                item.sockets.sockets &&
+                item.sockets.sockets[i] &&
+                item.sockets.sockets[i].plugOptions.length === 1
+            );
+        }
+
+        const curated =
+          item.season > 3 &&
+          (item.bucket && item.bucket.sort === 'Weapons') &&
+          item.tier.toLowerCase() === 'legendary' &&
+          item.masterworkInfo &&
```

Comment:
This seems to contradict the next comment?

Reply:
The comment should say `fully` masterworked, I'll update it